src/Kernel/Protocol.class.st

```diff
@@ -38,6 +64,17 @@ Protocol >> addMethodSelector: aSymbol [
 	^ methodSelectors add: aSymbol
 ]
 
+{ #category : #private }
+Protocol >> canBeRemoved [
+	^ self isEmpty
```

Comment:
canBeRemoved feels superfluous

Reply:
I'll remove it. I hesitated to remove it already and seen you think the same is enough for me to remove it :)